src/_damon_sysfs.py

```diff
@@ -74,7 +74,8 @@ def update_schemes_tried_bytes(kdamond_idxs):
 def update_schemes_tried_regions(kdamond_idxs):
     err = __write_state_file(kdamond_idxs, 'update_schemes_tried_regions')
     if err != None:
-        err = '%s (maybe schemes_tried_regions not supported?)' % err
+        # schemes_tried_regions feature available from mainline kernel v6.2 only
+        err = '%s (DAMON feature \'schemes_tried_regions\' is supported only from kernel version 6.2)' % err
```

Comment:
`__wite_state_file()` writes the keyword to the `state` file.  Writing to `state` file can fail if the there is a concurrent thread that doing another write already.  Hence, the reason of the failure could also be the concurrent writes.  What about keeping the original message but appending this hint? ("maybe schemes_tried_regions not supported?  AMON feature \'schemes_tried_regions\' is supported only from kernel version 6.2")

Reply:
ok got it, in that case, I was thinking of checking the current running kernel version and throwing this error (`DAMON feature \'schemes_tried_regions\' is supported only from kernel version 6.2`) only in cases when kernel version if < 6.2. In all other cases, we will just throw error (`updating schemes tried regions fail: writing update_schemes_tried_regions to /sys/kernel/mm/damon/admin/kdamonds/0/state failed ([Errno 22] Invalid argument)`) we receive from `__write_state_file `.

My idea is to not leave a question (`maybe schemes_tried_regions not supported?`) for customer, what do you say about this approach?

Reply:
I think removing the question is a nice approach.  Nonetheless, some distros provide kernels that having DAMON future features backported.  For example, AL2 provides v5.10 based kernel that having v6.7's DAMON code.  So, the error code may better to be adjusted for such cases.

I think we can use `feature_supported()` function to check if the running kernel is supporting the `update_schemes_tried_regions` and provide an error message.

Reply:
Sure SJ, I'll use `feature_supported()` to check if the running kernel supports `update_schemes_tried_regions ` and provide error message accordingly.

Reply:
Thanks, looking forward to the next revision!

Reply:
Sent a revision.
Used `feature_supported ()` to check if the feature is supported and accordingly throwing error message.